spyder/plugins/debugger/widgets/main_widget.py

```diff
@@ -438,7 +440,8 @@ def update_actions(self):
                 action = self.get_action(action_name)
                 action.setEnabled(pdb_prompt)
 
-            self._set_visible_control_debugger_buttons(pdb_prompt)
+            self._set_visible_control_debugger_buttons(pdb_prompt or
+                                                       is_debugging)
```

Comment:
```suggestion
            self._set_visible_control_debugger_buttons(
                pdb_prompt or is_debugging
            )
```

Small style suggestion.